account/test/integration/dbConnect.test.js

```diff
@@ -0,0 +1,29 @@
+import { expect, describe, it } from "@jest/globals";
+import db from "../../src/config/dbConnect.js";
+
+const STATUS = {
+	0: "Disconectado",
+	1: "Conectado",
+	2: "Conectando",
+	3: "Disconectando",
+};
+
+describe("Test Connection DB", () => {
+	it.only("MongoDB Connection", async () => {
+		async function isConnected(){
```

Comment:
Métodos que começam com "is" em geral são utilizados para retornar um valor booleano, neste caso, seria melhor dar um nome mais significativo ao método.